public/main.css

```diff
@@ -0,0 +1,157 @@
+/*****Header*****/
+
+.header {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-rows: 250px;
+    background-color: #a9d2de;
+}
+
+#fish {
+    display: grid;
```

Comment:
Here, since the element with an id of fish is the is the element that you are placing inside the header, which is a grid, you don't need to use the ```display: grid``` rule. Just setting the grid-column rule will work here.